frontend/src/components/layout/UserDropDownContainer.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import UserDropDown from './UserDropDown'
+
+interface Props {
+  user?: {
+    role?: string,
+    unit?: string,
+    department?: string,
+    group?: string
+  };
+  avatarUrl?: string;
+  userName?: string;
+}
+
+const UserDropDownContainer: React.FC<Props> = ({ user, avatarUrl, userName }) => {
+
+  const [open, setUserOpen] = React.useState(false);
```

Comment:
Could you please rename some variables to some more clear?
```suggestion
  const [isDropdownOpen, setDropdownState] = React.useState(false);
```